src/cmds/install.js

```diff
@@ -165,9 +165,28 @@ exports.handler = async function handler(argv) {
           fatal("Service manifest is not a well-formed JSON file.");
           break;
         case errors.ERROR_INVALID_SERVICE_MANIFEST:
-          error("Service manifest rejected due to errors:");
-          error(e);
-          process.exit(1);
+          fatal(`Service manifest rejected due to errors:\n\n${e.message}`);
```

Comment:
Passing errors to the `error` logger resulted in unsightly client-side stacktraces which don't help anyone. So let's just use fatal instead.